packaging/systemd/cloudstack-usage.service

```diff
@@ -24,7 +24,9 @@ After=network.target network-online.target
 [Service]
 Type=simple
 EnvironmentFile=/etc/default/cloudstack-usage
-ExecStart=/usr/bin/java $JAVA_OPTS -cp $CLASSPATH $JAVA_CLASS
```

Comment:
Put this and this would fix the pid issue:
```
Environment=JAVA_PID=$$
ExecStart=/usr/bin/java $JAVA_DEBUG -Dpid=$JAVA_PID $JAVA_OPTS -cp $CLASSPATH $JAVA_CLASS
```

Reply:
Comment: this would need to be fixed for 4.11 as well, let me send a quick fix instead.